src/hooks/use-filtered-market-coins.hook.ts

```diff
@@ -0,0 +1,63 @@
+import { useMemo, useState, useCallback } from 'react';
+import { useDispatch } from 'react-redux';
+
+import { MarketCoinsSortFieldEnum } from '../enums/market-coins-sort-field.enum';
+import { selectSortValue } from '../store/market/market-actions';
+import {
+  useFavouriteTokens,
+  useMarketTopCoinsWithoutTez,
+  useSortFieldSelector
+} from '../store/market/market-selectors';
+import { isString } from '../utils/is-string';
+import { sortMarketCoins } from '../utils/market.util';
+import { MarketCoin } from './../store/market/market.interfaces';
+
+export const useFilterdMarketCoins = () => {
```

Comment:
in this hook you are calling 1 thing with 3 different names: `tokens`, `coins`, `assets`.  Fix this